test/dummy/config/puma.rb

```diff
@@ -4,7 +4,7 @@
 # the maximum value specified for Puma. Default is set to 5 threads for minimum
 # and maximum; this matches the default thread size of Active Record.
 #
-max_threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
+max_threads_count = ENV.fetch("RAILS_MAX_THREADS", 5)
```

Comment:
I remember this fix or the one below it with PORT 3000 caused me some issues in another project I was working on, so we might need to double check or ignore these lines before merging the PR.